source/documentation/additional-information.html.md

```diff
@@ -339,17 +348,17 @@ After submitting the above losses, customers can view their loss position. If th
 
 ## Capital Gains Tax
 
-Capital Gains Tax is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
+Capital Gains Tax (CGT) is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
 
 * residential property
 * other property 
 * listed and unlisted shares and securities 
 
-For more information on capital gains tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
+For more information on Capital Gains Tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
```

Comment:
information about Capital Gains Tax

'information on' is associated with US English

Reply:
I can't find anything to support that 'information on' is US English, seems like it is fine in British English. Also that is not part of the change that's being reviewed.